src/engine/core/Camera.cpp

```diff
@@ -5,11 +5,11 @@
 namespace lei3d
 {
 	Camera::Camera(GLFWwindow* window, float yaw, float pitch, LookMode mode)
-		: m_Yaw{yaw}, m_Pitch{pitch}, m_LookMode{mode}
+		: m_Yaw{ yaw }, m_Pitch{ pitch }, m_LookMode{ mode }
 	{
 		m_FOVDeg = 45.0f;
 		m_NearPlane = 0.1f;
-		m_FarPlane = 700.0f;
+		m_FarPlane = 1400.0f;
```

Comment:
This is so we can see more terrain I'm guessing?

Reply:
yep yep, couldn't see the whole scene